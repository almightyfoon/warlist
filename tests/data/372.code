c9501b444xcQcvcv7y6yfs5U6767__fk6r4x6S4C6u6u6H5Rft5Rft5Rft