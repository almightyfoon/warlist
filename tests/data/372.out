https://conflictchamber.com/#c9501b444xn0cvcv7y6yfs5U6767__fk6r4x6S4C6u6uncn9n1n9n1n9n1

Skorne Army - 75 / 75 points

[Makeda 2] Supreme Archdomina Makeda [+27]
 - Basilisk Krea [7]
 - Despoiler [17]
Mortitheurge Willbreaker [4]
Mortitheurge Willbreaker [4]
Orin Midwinter, Rogue Inquisitor [5]
Tyrant Rhadeim [9]
Legends of Halaak [8]
Paingiver Beast Handlers (min) [5]
Praetorian Ferox (max) [20]
Praetorian Ferox (max) [20]

Skorne Army - 65 / 75 points

!!! This army must contain at least 99 total points.

[Zaal 2] Zaal, the Ancestral Advocate [+29]
 - Agonizer [6]
 - Basilisk Krea [7]
 - Cyclops Raider [9]
 - Cyclops Shaman [8]
Ancestral Guardian [5]
Ancestral Guardian [5]
Hakaar the Destroyer [6]
Immortals (max) [13]
 - Extoller Advocate [3]
Immortals (max) [13]
 - Extoller Advocate [3]
Immortals (max) [13]
 - Extoller Advocate [3]
